Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/VisualToastNotificationBasicForm.cs

```diff
@@ -53,27 +53,33 @@ public VisualToastNotificationBasicForm(KryptonBasicToastNotificationData data)
 
         private void UpdateText()
         {
-            kwlblContent.Text = _basicToastNotificationData.NotificationContent;
+            kwlblContent.Text = _basicToastNotificationData.NotificationContent ?? null;
 
             kwlblHeader.Text = _basicToastNotificationData.NotificationTitle;
 
             kwlblHeader.TextAlign =
                 _basicToastNotificationData.NotificationTitleAlignment ?? ContentAlignment.MiddleCenter;
         }
 
-        private void UpdateFadeValues()
-        {
-            FadeValues.FadingEnabled = _basicToastNotificationData.UseFade ?? false;
-        }
+        private void UpdateFadeValues() => FadeValues.FadingEnabled = _basicToastNotificationData.UseFade ?? false;
```

Comment:
Why is `UseFade` nullable, just default to false !

Reply:
`UseFade` is now false by default